settings.py

```diff
@@ -185,7 +186,10 @@
     'django.core.context_processors.i18n',
 ]
 
-TEMPLATE_DIRS = []
+location = lambda x: os.path.join(os.path.dirname(os.path.realpath(__file__)), x)
```

Comment:
might want to base this on `BASE_DIR`, which is defined above. Also, what do you think of changing the name to `_location`? Another alternative would be to inline it below since it appears to be referenced only once.